File: ABC088B.js
'use strict'

function Main(input){
  const tmpInputs = input.toString().split("\n");
  const N = parseInt(tmpInputs[0]);
  const aArray = tmpInputs[1].toString().split(" ").map(Ai => parseInt(Ai));
  const sortedArray = aArray.sort((a,b)=> b-a);

  let AliceScore = 0;
  let BobScore = 0;

    for(let currentTurnCount=1; currentTurnCount <= N; currentTurnCount++){
      currentTurnCount % 2 ? AliceScore += sortedArray[currentTurnCount-1] : BobScore += sortedArray[currentTurnCount-1];
    }

  console.log(AliceScore-BobScore);
}

//*この行以降は編集しないでください（標準入出力から一度に読み込み、Mainを呼び出します）
Main(require("fs").readFileSync("/dev/stdin", "utf8"));

File: ABC088B.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"sort"
	"strconv"
)

var sc = bufio.NewScanner(os.Stdin)

func nextInt() int {
	sc.Scan()
	i, e := strconv.Atoi(sc.Text())
	if e != nil {
		panic(e)
	}
	return i
}

func main() {
	sc.Split(bufio.ScanWords)

	N := nextInt()
	cardArray := []int{}

	//スライスに2行目の各要素をappend
	for i := 0; i < N; i++ {
		cardArray = append(cardArray, nextInt())
	}
	// 1.8のメソッド…
	// sort.Slice(cardArray, func(i, j int) bool { return cardArray[i] > cardArray[j] })
	sort.Sort(sort.Reverse(sort.IntSlice(cardArray)))

	var AliceScore, BobScore int

	for currentTurnCount := 1; currentTurnCount <= N; currentTurnCount++ {
		if currentTurnCount%2 == 1 {
			AliceScore += cardArray[currentTurnCount-1]
		} else {
			BobScore += cardArray[currentTurnCount-1]
		}
	}

	fmt.Print(AliceScore - BobScore)
}
